src/test/java/org/opentripplanner/transit/model/timetable/TripTimesTest.java

```diff
@@ -0,0 +1,84 @@
+package org.opentripplanner.transit.model.timetable;
+
+import java.util.Collection;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.opentripplanner.framework.i18n.I18NString;
+import org.opentripplanner.framework.i18n.NonLocalizedString;
+import org.opentripplanner.model.StopTime;
+import org.opentripplanner.transit.model._data.TransitModelForTest;
+import org.opentripplanner.transit.model.framework.Deduplicator;
+
+class TripTimesTest {
```

Comment:
We already have a `TripTimesTest` but it's in the wrong package.

Can you move it here and rename it `TripTimesUpdateTest`?

Reply:
With a bit of refactoring, it was actually quite easy to merge both test classes.

Reply:
Thanks for that.

Could I perhaps ask for one more round of refactoring?

- remove the @BeforeAll from `createInitialTripTimes` and simply call the method in those tests that actually need it
- move your headsign tests into a nested class with the @Nested annotation, [like this](https://github.com/opentripplanner/OpenTripPlanner/blob/ba1f03cb1b2162e747617e228ac5d6277d650f7b/src/test/java/org/opentripplanner/street/model/edge/RentalRestrictionExtensionTest.java#L178-L192)

I know it's a bit nitpicky but I like well structured tests.

Reply:
Will look into it.